src/exampleTypes/ResultTable.js

```diff
@@ -127,10 +127,12 @@ let Header = withStateLens({ popover: false, adding: false, filtering: false })(
       disableFilter,
       disableSort,
       field,
+      sortBy,
       label,
       hideMenu,
       typeDefault,
     } = fieldSchema
+    sortBy = sortBy ? `${field}.${sortBy}` : field
```

Comment:
Let's simplify this to just use sortyBy instead of prefixing with field. It'll be more flexible and easier to understand.

Let's also call it `sortField` instead of `sortBy` to be consistent with the results type